scripts/onboarding/aks/onboarding-using-msi-auth/existingClusterParam.json

```diff
@@ -17,6 +17,9 @@
       "enableSyslog": {
         "value": false
       },
+      "enableContainerLogV2": {
+        "value": true
```

Comment:
we need to update our  template onboarding docs about the ContainerLogV2 default.